### PR TITLE
Improve performance

### PR DESCRIPTION
**Pull request by thomaswelter**
This PR adds a symbol cache to improve performance in larger files.
Thank you for your plugin, very useful!

### CODE REVIEW

File: move_by_symbols.py [hunk 1]
```diff
@@ -13,6 +13,12 @@
 else:
     from symbols_highlighting import add_highlighting
 
+symbolCache = {}
+
+class MoveBySymbolsEventListener(sublime_plugin.EventListener):
+    def on_modified_async(self, view):
+        if view.id() in symbolCache:
+            del symbolCache[view.id()]
```
Comment (abusalimov) [inline move_by_symbols.py:22]: Shouldn't this also be invalidated when the `view` is closed completely?

**Comment by abusalimov** [comment]
Thanks for the contribution!
Frankly, I don't use Sublime anymore, nor do I maintain the plugin. I'm not against merging the PR though, so thanks again!

**Comment by thomaswelter** [comment]
It doesn't matter to me as I forked the code locally but it might be nice for other users. I'll let you decide whether you want to merge or not.

File: move_by_symbols.py [hunk 2]
```diff
@@ -15,11 +15,20 @@
...
 symbolCache = {}
 
+# remove symbol cache on edit or on close
 class MoveBySymbolsEventListener(sublime_plugin.EventListener):
     def on_modified_async(self, view):
-        if view.id() in symbolCache:
+        if ST3 and view.id() in symbolCache:
+            del symbolCache[view.id()]
+
+    def on_modifies(self, view):
```
Comment (abusalimov) [inline move_by_symbols.py]: This looks suspiciously like a typo. Is it really called `on_modifies` with the `s` in the end?

**Comment by abusalimov** [comment]
Thank you @thomaswelter! I created a new tag v1.1.0 including this change.

**Pull request merged**